spec/controllers/v1.0/requests_controller_spec.rb

```diff
@@ -81,7 +81,7 @@
   end
 
   # Test suite for GET /requests?state=
-  describe 'GET /requests?state=notified' do
+  describe 'GET /requests?filter[state]=notified' do
```

Comment:
In the future these unrelated changes would preferable be moved to a separate PR or at least a separate commit as it is unrelated to the other work in this PR.